Nodejs/Product/Nodejs/Workspace/PackageJsonTestContainerDiscoverer.cs

```diff
@@ -66,20 +66,26 @@ private async Task AttemptUpdateAsync()
             if (workspace != null)
             {
                 var indexService = workspace.GetIndexWorkspaceService();
-                var filesDataValues = await indexService.GetFilesDataValuesAsync<string>(NodejsConstants.TestRootDataValueGuid);
-
-                lock (this.containerLock)
+                // This returns null for LiveShare workspace so adding a check.
```

Comment:
Can you please add a comment about the consequences of skipping the work below?  I believe it just means no tests will be discovered and the Test Explorer will be empty?

Reply:
That's correct. Will add!